### DISCUSSION TITLE
Improving package setup

### DISCUSSION THREAD

**Discussion by ndaelman-hu** [FAIRmat-NFDI/nomad-simulations#74] [Ideas]
So, going over the current version of the `README.md`, I have a question about the following step:

```bash
pip install '.[dev]' --index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple
```

After following this step, I was still encountering issues with the `$PYTHONPATH`. I found a pretty elegant solution from this [thread](https://stackoverflow.com/questions/4757178/how-do-you-set-your-pythonpath-in-an-already-created-virtualenv), top answer no. 2 (not 1). This approach is very clean, as it confines the paths to the environment without having to install other utilities. The first suggestion, meanwhile, will end up affecting the global `$PYTHONPATH` too (until a reboot at least). That's frankly unacceptable.

@JosePizarro3 I suggest we add a line about this. Before opening an issue, I 'd like to know what other reasons the aforementioned command has.

**Comment by JosePizarro3** [comment]
Which issues do you mean? At that point, you don't have anything on the PYTHONPATH.

**Reply by ndaelman-hu** [reply]
Mmm, that doesn't sound completely right. When I `pip` install anything, the bindings (or the executables themselves) have to be placed under one of the `$PYTHONPATH`s. Creating a new environment, should set a new path, to avoid any name conflicts. You can tell that the `activate` script sets a new `$PYTHONHOME`, etc. and unsets it upon deactivation.

Still, certain paths weren't showing up, so I went with the other option.

**Reply by JosePizarro3** [reply]
I am not sure if I understand how you installed this. Maybe we can sit together and check it together.

**Comment by JosePizarro3** [comment]
Regarding adding that command so that PYTHONPATH is added to the venv, indeed, we could even have it as a shell script, maybe we can talk with Hampus about it? 🙂

**Reply by ndaelman-hu** [reply]
Setting up a script for the `$PYTHONPATH` is indeed simple enough. It should just take a couple of params
